gradio/blocks.py

```diff
@@ -267,6 +269,9 @@ def set_event_trigger(
         elif every:
             raise ValueError("Cannot set a value for `every` without a `fn`.")
 
+        if event_name == "change":
+            trigger_mode = "multiple"
```

Comment:
This will override the `trigger_mode` even if a user wants to pass in a different one for `.change()`. I'd suggest setting the default value of `trigger_mode` to `None` and then if its still `None` here, setting it to "always_last" (not "multiple") for `change` and "once" for everything else

Reply:
Oh yeah, I changed it to "multiple" for testing, forgot to change it back